packages/sdk/chainDeploy/mainnets/polygon.ts

```diff
@@ -94,6 +95,28 @@ export const deployConfig: ChainDeployConfig = {
       underlying: assets.find((a) => a.symbol === assetSymbols["JSGD-XSGD"])!.underlying,
       otherParams: ["0x18DAdac6d0AAF37BaAAC811F6338427B46815a81", "10"],
     },
+    {
+      // PAR/USDC
+      strategy: "ArrakisERC4626",
+      name: "PARUSDC",
+      underlying: assets.find((a) => a.symbol === assetSymbols["arrakis_USDC_PAR_005"])!.underlying,
+      otherParams: [
+        "0x528330fF7c358FE1bAe348D23849CCed8edA5917", // arrakis mimo pool
+        "", // rewardsDestination
```

Comment:
Why is this empty?

Reply:
I don't know what would the destination but we can set the destination after contract is deployed. And I took a look at the dotdot deployed in bsc and it's giving empty there as well.

Reply:
actually, remove this config